scripts/source-maintenance.sh

```diff
@@ -616,12 +616,12 @@ printRawAmFile ()
     echo "## $GeneratedByMe"
     echo
 
-    echo -n "$1 ="
+    printf "%s =" "$1"
     # Only some files are formed from *.po filenames, but all such files
     # should list *.lang filenames instead.
     git ls-files $2$3 | sed -e s%$2%%g -e 's%\.po%\.lang%g' | while read f; do
         echo " \\"
-        echo -n "    ${f}"
+        printf "    %s" "${f}"
```

Comment:
These lines are just supposed to produce folded-whitespace. Now that we are using `printf` here the initial `echo` can be merged into the format string properly and that weird extra line elided.

```
    printf " \\\n\t%s" "${f}"
```

Reply:
FWIW, I find the current PR code (two code lines producing two output lines) more readable than the proposed single-line code that prints two lines.

Reply:
I agree. @yadij is changing this a dealbreaker for you?

Reply:
I believe that @rousskov already mistaking this as adding **two** lines of output is a good evidence of why this should **not** be done as two statements.

This code loop is actually only printing one line in the yoda-style form: `line-terminator line-text`

If this code were producing a normal text list for humans this code would look like:
 . `printf "\n%s" "${f}"`

However, automake build tools require folded-whitespace lines. Which have a fancy terminator:
 . ` line := *CHAR ( SP '\' LF HTAB )`
instead of the usual
 . ` line := *CHAR ( *CR LF )`

And thus we end up with:
 . `printf " \\\n\t%s" "${f}"`

Reply:
> I believe that @rousskov already mistaking this as ...

Again and again, please do not review people.

FWIW, I know what this code does and (still) think that two code lines are better than one in this case. Furthermore, since this PR does not have to modify the first `echo` line at all, one can classify this change request itself (and, hence, the discussion of its merits) to be outside this PR scope.

If you do want to polish this code in this PR, then adding comments (to clarify that the first code line terminates the "current" output line and the second code line starts the "next" output line) may be a good idea, but it may more than one iteration to get those comments right.

Reply:
The reason for my request is to remove an old hack which is no longer necessary. If you are going to insist on splitting the line terminator across multiple lines. Why not print three lines? that makes as much sense as insisting a hack remains untouched. IMO the fix both easy and relevant to the current PR.

Reply:
The tab had been expanded in translation sometime, and was showing up as a number of spaces.
printf "\t" is clearer, so switching to that.
For the sake of consistency, I'll also turn the terminating `echo ""` into a printf '\n'